File: 03_soccer_team_points/ocaml/bin/main.ml
let points_for_win = 3
let points_for_loss = 0
let points_for_tie = 1

let () = print_endline "\n--PROGRAMA PARA CALCULAR PUNTOS DE UN EQUIPO DE FUTBOL--\n"

let () = print_endline "Ingrese el nombre del equipo: "
let team_name = read_line ()

let () = print_endline (Printf.sprintf "Ingrese el número de partidos ganados por el equipo %s: " team_name)
let wins = int_of_string (read_line ())

let () = print_endline (Printf.sprintf "Ingrese el número de partidos perdidos por el equipo %s: " team_name)
let losses = int_of_string (read_line ())

let () = print_endline (Printf.sprintf "Ingrese el número de partidos empatados por el equipo %s: " team_name)
let ties = int_of_string (read_line ())

let points = (wins * points_for_win) + (losses * points_for_loss) + (ties * points_for_tie)

let () = print_endline (Printf.sprintf "\nEl equipo %s tiene un total de %d puntos" team_name points)

File: 03_soccer_team_points/java/src/Main.java
import java.util.Scanner;

public class Main {
    private static final int POINTS_FOR_WIN = 3;
    private static final int POINTS_FOR_LOST = 0;
    private static final int POINTS_FOR_TIE = 1;

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("--PROGRAMA PARA CALCULAR PUNTOS DE UN EQUIPO DE FUTBOL--\n");

        System.out.println("Ingrese el nombre del equipo:");
        String team_name = input.nextLine();

        System.out.println(String.format("Ingrese la cantidad de partidos ganados por %s:", team_name));
        int win = input.nextInt();

        System.out.println(String.format("Ingrese la cantidad de partidos perdidos por %s:", team_name));
        int lost = input.nextInt();

        System.out.println(String.format("Ingrese la cantidad de partidos empatados por %s:", team_name));
        int ties = input.nextInt();

        int total = win * POINTS_FOR_WIN + lost * POINTS_FOR_LOST + ties * POINTS_FOR_TIE;

        System.out.println("\nEl equipo " + team_name + " tiene un total de " + total + " puntos");
    }
}